Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -221,17 +221,17 @@ public class CRTClientEngine: HTTPClient {
             // END - smithy.client.http.requests.queued_duration
 
             // TICK - smithy.client.http.connections.limit
```

Comment:
`httpMetricsUsage` is now an actor, so changes to it must be made with an async method call, not a synchronous setter.